content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,64 @@
+# Machine Learning Model Registry
+
+A **model registry** is a repository to catalog ML models and their versions.
+Artifacts from your data science projects can be discovered, audited, shared,
+tested, and deployed from here.  
+DVC works on top of Git to enable these capabilities on your existing software
+engineering stack. Other tools from [Iterative](https://iterative.ai/) provide
+complementary features and more usability.
+
+![](/img/ml_model_registry_placement.jpg) _A common placement for model
+registries is between development and applications. See also [Data Registries]._
+
+[data registries]: /doc/use-cases/data-registries
+
+Model registries give your team key collaborative capabilities:
+
+- Store and present your model artifacts to others, from early-stage
```

Comment:
just models  ... not need to introduce additional terms, they don't give any additional information?